content/en/blog/_posts/2023-04-06-seccomp-profiles-edge.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: blog
+title: "Having fun with seccomp profiles on the edge"
+date: 2023-04-06
+slug: seccomp-profiles-edge
+---
+
+**Author**: Sascha Grunert
+
+The [Security Profiles Operator (SPO)][spo] is a feature-rich operator for
+Kubernetes to make managing seccomp, SELinux and AppArmor profiles easier than
+ever. Recording those profiles from scratch is one of the key features of this
+operator, which usually involves the integration into large CI/CD systems. Being
+able to test the recording capabilities of the operator in edge cases is one of
+the recent development efforts of the SPO and makes it excitingly easy to play
+around with seccomp profiles.
+
+[spo]: https://github.com/kubernetes-sigs/security-profiles-operator
+
+## Recording seccomp profiles with `spoc record`
+
+The [latest release][spo-latest] of the Security Profiles Operator ships a new
+command line interface called `spoc`, which is a little helper tool for
+recording and replaying seccomp profiles among various other things which are
+out of scope of this blog post.
+
+[spo-latest]: https://github.com/kubernetes-sigs/security-profiles-operator/releases/latest
```

Comment:
Most important change is to be specific about which release, as https://github.com/kubernetes-sigs/security-profiles-operator/releases/latest is not a permalink.

Is this right?
```suggestion
The [v0.7.1](https://github.com/kubernetes-sigs/security-profiles-operator/releases/tag/v0.7.1)
of the Security Profiles Operator shipped a new command line interface called `spoc`, a little
helper tool for recording and replaying seccomp profiles amongst various other things that
are out of scope of this blog post.
```

Reply:
:+1: